ntbs-ui-tests/Steps/Steps.cs

```diff
@@ -1,228 +1,485 @@
-
-        [Then(@"I should see all submission error messages")]
-        public void ThenIShouldSeeAllSubmissionErrorMessages()
-        {
-            var summaryText = Browser.FindElement(By.ClassName("nhsuk-error-summary")).Text;
-            Assert.Contains("Notification date is a mandatory field", summaryText);
-            Assert.Contains("Hospital is a mandatory field", summaryText);
-            Assert.Contains(ValidationMessages.DiseaseSiteIsRequired, summaryText);
-            Assert.Contains("Diagnosis date is a mandatory field", summaryText);
-            Assert.Contains("Date of birth is a mandatory field", summaryText);
-            Assert.Contains("Sex is a mandatory field", summaryText);
-            Assert.Contains("Postcode is a mandatory field", summaryText);
-            Assert.Contains("Birth country is a mandatory field", summaryText);
-            Assert.Contains("Given name is a mandatory field", summaryText);
-            Assert.Contains("NHS number is a mandatory field", summaryText);
-            Assert.Contains("Family name is a mandatory field", summaryText);
-            Assert.Contains("Ethnic group is a mandatory field", summaryText);
-        }
-
-        [Then(@"The notification should be denotified")]
-        public void ThenCurrentNotificationShouldBeDenotified()
-        {
-            var notificationHeading = Browser.FindElement(By.TagName("h1")).Text;
-            Assert.Contains("Denotified", notificationHeading);
-        }
-
-        private void SaveNotificationInDatabase(Notification notification)
-        {
-            var options = new DbContextOptionsBuilder<NtbsContext>();
-            options.UseSqlServer(Settings.EnvironmentConfig.ConnectionString);
-            using (var context = new NtbsContext(options.Options))
-            {
-                context.Add(notification);
-                context.SaveChanges();
-            }
-            TestContext.AddedNotificationIds.Add(notification.NotificationId);
-        }
-
-        private int GetUserIdFromUsername(string username)
-        {
-            var options = new DbContextOptionsBuilder<NtbsContext>();
-            options.UseSqlServer(Settings.EnvironmentConfig.ConnectionString);
-            using (var context = new NtbsContext(options.Options))
-            {
-                return context.User.Single(u => u.Username.ToLower() == username.ToLower()).Id;
-            }
-        }
-    }
-}
+﻿using System;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading;
+using Microsoft.EntityFrameworkCore;
+using ntbs_service.DataAccess;
+using ntbs_service.Helpers;
+using ntbs_service.Models.Entities;
+using ntbs_service.Models.Validations;
+using ntbs_ui_tests.Helpers;
+using ntbs_ui_tests.Hooks;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using TechTalk.SpecFlow;
+using Xunit;
+
+namespace ntbs_ui_tests.Steps
+{
+    [Binding]
+    public class Steps
```

Comment:
This class is getting quite large, to say the least! Have you looked into ways to break it up at all? It would be good if there was a logical structure so that devs would have an idea of where a particular step would live.